docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -577,11 +577,11 @@ Some examples of high contrast changes include:
 
   Before:
 
-  ![Disabled text before accessibility improvements](./media/wf-disabled-before.png) 
+  ![Screenshot of disabled text before accessibility improvements.](./media/whats-new-in-accessibility/wf-disabled-menu-item-before.png) 
```

Comment:
```suggestion
  ![Screenshot of an app that uses different controls running in high contrast mode before accessibility improvements.](./media/whats-new-in-accessibility/wf-disabled-menu-item-before.png) 
```

Reply:
You could call this image something to indicate high-contrast-mode-before and then the other one after.